src/workflows/airqo_etl_utils/storage/cloud_storage.py

```
@@ -2,6 +2,10 @@
 import logging
 import os
 from tempfile import NamedTemporaryFile
+
+from pkg_resources import safe_name
+
+from pkg_resources import safe_name
```

Ruff (0.15.0)

[error] Redefinition of unused `safe_name` from line 6: `safe_name` redefined here

Remove definition: `safe_name`

(F811)